source/brailleDisplayDrivers/braillePen.py

```diff
@@ -0,0 +1,143 @@
+#brailleDisplayDrivers/braillePen.py
+#A part of NonVisual Desktop Access (NVDA)
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+#Copyright (C) 2014 Harpo Sp. z o. o. <http://www.harpo.com.pl>
+
+import braille
+import queueHandler
+from logHandler import log
+from ctypes import *
+from ctypes.wintypes import *
+import time
+import config
+import inputCore
+import hwPortUtils
+import brailleInput
+import wx
+import hwIo
```

Comment:
nit: Quite a few unneeded imports here: queueHandler, ctypes, ctypes.wintypes, time, config, wx. Please remove them. You also probably won't need log; see below.